international-labour-organization.csl

```diff
@@ -78,9 +81,10 @@
     </choose>
   </macro>
   <macro name="publisher">
-    <group delimiter=", ">
+    <group>
       <text variable="event"/>
-      <text variable="publisher-place" prefix="(" suffix=")"/>
+      <text variable="publisher-place" prefix=" (" suffix=", "/>
+      <text variable="publisher" suffix=")"/>
```

Comment:
@vl2, parentheses should not be split over multiple elements. Instead of

```xml
    <group>
      <text variable="event"/>
      <text variable="publisher-place" prefix=" (" suffix=", "/>
      <text variable="publisher" suffix=")"/>
    </group>
```

you can use

```xml
    <group delimiter=" ">
      <text variable="event"/>
      <group prefix="(" suffix=")" delimiter=", ">
        <text variable="publisher-place"/>
        <text variable="publisher"/>
      </group>
    </group>
```